### ISSUE TITLE
[BUG]: File picker looks broken

### ISSUE THREAD

**Issue by ivakam** [neuromorph/openbar#85]
**Describe the bug**
Title, pretty much. Something isn't right with the file picker theming.

**To Reproduce**
1. Set a color theme for shell/non-bar things
2. Do something that triggers a file picker like right click -> "save as" in a browser

**Relevant  Specs:**
 - Open Bar version: 41
 - Gnome version: 47
 - OS/Distribution: EndeavourOS

**Screenshots**
![image](https://github.com/user-attachments/assets/b57d9b6f-7ebc-4647-b4ca-643b829e922b)

**Comment by neuromorph** [comment]
Hello,

Are you using any Gtk theme outside of OpenBar? And please share your OpenBar settings for "Gtk / Flatpak Apps" tab.

The file picker is Gtk. There was supposed to be new update for it so maybe something changed recently but this issue is not expected.  

Thank you for reporting!

**Comment by ivakam** [comment]
Nope, no other GTK theming present. I'll attach the exported settings so you can toy around with it yourself (change extension to .theme): 
[gruvbox_light.txt](https://github.com/user-attachments/files/17774161/gruvbox_light.txt)

I've also noticed that other native GTK windows (such as settings menus and such) are whiter than I feel they should? It seems that the theming is not being applied quite as uniformly as is probably intended?

Thank you for replying!

**Comment by neuromorph** [comment]
Hello,

About the File picker:
I tried by importing your settings in my OpenBar and it worked OK. No issues like in your screenshot. Though, I am on Ubuntu 24.04 / Gnome 47. It is previously tested on Fedora 41 and Gnome OS so I don't expect issues there but I will check again since I have them in VM. If it is something to do with EndeavourOS, I am not sure.

> I've also noticed that other native GTK windows (such as settings menus and such) are whiter than I feel they should? It seems that the theming is not being applied quite as uniformly as is probably intended?

Please add any screenshots for where you see some mismatch, for clarity.

However, I am quite busy at the moment so it might take some time to figure it out. If you have any other system or something that you can try it on, please do so in the meanwhile. Feel free to add here any inputs that may help.

Thanks!
